canonical-websites/www.ubuntu.com
# Copy kubernetes docs directly

## Done
Pull the markdown from the upstrream source into the site

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to http://0.0.0.0:8001/kubernetes/docs
- Click through the pages and see they render

## Issue / Card
Fixes #4344
